page-template-cluster.php

```diff
@@ -57,20 +62,27 @@
 	<div class="container mt-4 mt-sm-5 mb-5 pb-sm-4">
 		<div class="row">
 			<div class="col-lg-8">
-				<?php the_content(); ?>
+				<section id="cluster-content" aria-label="Main Content">
+					<?php the_content(); ?>
+				</section>
 			</div>
 			<div class="col-lg-4">
 				<?php if ( $cluster_colleges && count( $cluster_colleges ) > 0 ) : ?>
-				<h2 class="h5 mb-4">UCF Colleges Involved in <?php echo $post->post_title; ?>:</h2>
-				<ul class="list-unstyled">
-				<?php foreach( $cluster_colleges as $college ) : ?>
-					<li class="mb-2">
-						<a href="<?php echo $college['college_url']; ?>" target="_blank">
-							<?php echo $college['college_name']; ?>
-						</a>
-					</li>
-				<?php endforeach; ?>
-				</ul>
+				<section id="cluster-colleges-centers" aria-labelledby="cluster-colleges-centers-heading">
+					<?php if ( $cluster_colleges_image ) : ?>
+					<img src="<?php echo $cluster_colleges_image['url']; ?>" class="img-fluid mb-4<?php echo $cluster_image_classes; ?>" alt="<?php echo $cluster_colleges_image['alt']; ?>">
+					<?php endif; ?>
+					<h2 id="cluster-colleges-centers-heading" class="h5 mb-4"><?php echo $cluster_colleges_heading; ?>:</h2>
+					<ul class="list-unstyled">
+					<?php foreach( $cluster_colleges as $college ) : ?>
```

Comment:
Is there ever going to be a situation where `$cluster_colleges` is empty?  (Do we need to add a empty check above the wrapper `<ul>` to avoid printing an empty list?)

Reply:
There is an if-null and count check around the entire section.. I just went ahead and moved it above/below the `col-lg-4` div to make sure it doesn't print an empty column if there aren't any colleges.